packages/venia-concept/src/index.css

```diff
@@ -116,3 +116,339 @@ button:disabled {
     cursor: default;
     touch-action: none;
 }
+
+/*
+ * Target all rows.
+ */
+:global(.venia-home-row-1),
+:global(.venia-home-row-2),
+:global(.venia-home-row-3) {
+    --brightBlue: 233 100% 056%;
+    max-width: 1440px !important;
+}
```

Comment:
We talked about this in review: is there a way to isolate the definition of these styles into a component that includes them but can then be `React.lazy` loaded?

Reply:
Fixed in 00985934825c34b616a03699470c94f4cc2c6887.